modules/mobile/app/controllers/mobile/v0/community_care_providers_controller.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    class CommunityCareProvidersController < ApplicationController
+      DEFAULT_PAGE_NUMBER = 1
+      DEFAULT_PAGE_SIZE = 10
+      RADIUS = 60 # default value used in web app
+      SPECIALTIES = {
+        primaryCare: %w[207QA0505X 363LP2300X 363LA2200X 261QP2300X],
```

Comment:
I chose to make these keys (which are the values of query params) camel case. I could've let the front-end send them as strings like "primary care", but I wasn't sure that was any better on their side.